game.go

```diff
@@ -354,6 +358,17 @@ func (p *Game) startLoad(fs spxfs.Dir, cfg *Config) {
 	p.windowHeight_ = cfg.Height
 }
 
+func (p *Game) canBindSprite(sprite Spriter, name string) bool {
```

Comment:
this param is useless

Reply:
ok. Done in pr 336.
https://github.com/goplus/spx/pull/336
The pr 335 contains other unexpected commit.
Use pr 336 instead.